packages/xo-server/src/xo-mixins/remotes.js

```diff
@@ -83,7 +83,23 @@ export default class {
 
   async testRemote(remote) {
     const handler = await this.getRemoteHandler(remote)
-    return handler.test()
+    const { readRate, writeRate, ...answer } = await handler.test()
+
+    if (answer.success) {
+      const benchmark = {
+        readRate,
+        timestamp: Date.now(),
+        writeRate,
+      }
+      await this._updateRemote(remote, {
+        benchmarks:
+          remote.benchmarks !== undefined
+            ? [...remote.benchmarks, benchmark]
```

Comment:
We should limit the number of stored benchmarks.

Reply:
5 max, it's enough ?

Reply:
I think we can leave a lot more, like 50.

I just want a limit to avoid keeping them forever and ending with thousands benchmarks on the object.